config/locales/en.yml

```
@@ -135,6 +137,8 @@ en:
       - A spark for every powder keg.
       - Esperanza no espera.
 
+    site_mode: Switch to the lite version of the site >>
```

```suggestion
    site_mode: Switch to the lite version of the site »
```

Let's use the fancy double chevron unicode.